frontend/src/scenes/App.tsx

```diff
@@ -19,6 +19,20 @@ import { router } from 'kea-router'
 import { CommandPalette } from 'lib/components/CommandPalette'
 import { UpgradeModal } from './UpgradeModal'
 import { teamLogic } from './teamLogic'
+import posthog from 'posthog-js'
+import * as Sentry from '@sentry/browser'
+
+posthog.init((window as any).JS_POSTHOG_API_KEY || 'sTMFPsFhdP1Ssg', { api_host: (window as any).JS_POSTHOG_HOST })
+
+if ((window as any).SENTRY_DSN) {
```

Comment:
I don't think this code belongs here (in a random component).

There's frontend/src/index.tsx which renders this component. Similar to `initKea` I'd extract a module and call this code there.